### PR TITLE
added faction filter

### PR DESCRIPTION
**Pull request by TempoLab**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -81,6 +81,14 @@ function App() {
     })
   }, [usersRevivedAsObject, parsedUserEventsAsObject])
 
+  const userDataAsAnArrayFilteredByFaction = useMemo(() => {
+    const factionSearchParam = factionFilterValue
```
Comment (chopfitzroy) [inline src/App.jsx]: NIT: Might as well make this:
```js
const factionSearchParam = factionFilterValue.toLocaleUpperCase();
```
That way you don't have to do that later and it makes a bit. more sense as to why this gets it's own assignment.

**Review by chopfitzroy — DISMISSED** [review]
Approved, you can choose to make the suggested change or not.

File: src/App.jsx (hunk 1 above)
Comment (TempoLab) [reply src/App.jsx]: Makes total sense, change made.

**Review by chopfitzroy — APPROVED** [review]
Great work 😘

**Pull request merged**